src/app/main/subsysmgr.c

```diff
@@ -302,7 +302,8 @@ subsystems_dump_list(void)
 {
   for (unsigned i = 0; i < n_tor_subsystems - 1; ++i) {
     const subsys_fns_t *sys = tor_subsystems[i];
-    printf("% 4d\t%s\n", sys->level, sys->name);
+    printf("% 4d\t%16s\t%s\n", sys->level, sys->name,
+           sys->location?sys->location:NULL);
```

Comment:
NULL works on some systems, because the bytes there are '\0'. But it's not clear from man page that `printf("%s", NULL);` is guaranteed to work.

```suggestion
           sys->location?sys->location:"");
```